packages/zoe/src/contractFacet/types.js

```diff
@@ -28,6 +28,10 @@
  * @property {MakeZCFMint} makeZCFMint
  * @property {(exit: ExitRule=) => ZcfSeatKit} makeEmptySeatKit
  * @property {SetTestJig} setTestJig
+ * @property {() => Promise<AdminNode>} getAdminNode - The AdminNode can be
+ * used to shut down the vat or to find out when it has been shut down. To make
+ * its use visible, it can only be retrieved from zcf. So if a contract is
+ * making use of this functionality it will be visible in the contract's source.
```

Comment:
No, we don't want to expose this authority if it means that a vat can be shutdown without actually calling the `shutdown` method.

Reply:
That's a strong argument. I'll switch to exposing a facet that doesn't include terminate.